cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -103,31 +104,34 @@ class FiniteElement
   /// @return The string of the finite element family
   std::string family() const noexcept;
 
-  /// Evaluate all derivatives of the basis functions up to given order
-  /// at given points in reference cell
-  /// @param[in,out] values Four dimensional xtensor that will be filled
-  /// with the tabulated values. Should be of shape {num_derivatives,
-  /// num_points, num_dofs, reference_value_size}
-  /// @param[in] X Two dimensional xtensor of shape [num_points,
-  /// geometric dimension] containing the points at the reference
-  /// element
+  /// @brief Evaluate derivatives of the basis functions up to given order
+  /// at points in the reference cell.
+  /// @param[in,out] values Array that will be filled with the tabulated
+  /// basis values. Must have shape `(num_derivatives, num_points,
+  /// num_dofs, reference_value_size)` (row-major storage)
+  /// @param[in] X The reference coordinates at which to evaluate the
+  /// basis functions. Shape is `(num_points, geometric dimension)`
```

Comment:
Shouldn't this actually be the topological dimension of the underlying cell? As the geometric dimension of a triangular manifold is 3, while these are reference coordinates, which should be dim 2.